skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkiaLayer.jvm.kt

```diff
@@ -275,17 +284,27 @@ actual open class SkiaLayer internal constructor(
 
     open fun dispose() {
         check(isEventDispatchThread()) { "Method should be called from AWT event dispatch thread" }
-
         if (isInited && !isDisposed) {
-            redrawer?.dispose()  // we should dispose redrawer first (to cancel `draw` in rendering thread)
-            contextHandler?.dispose()
-            picture?.instance?.close()
-            pictureRecorder.close()
+            disposeSkiaResources()
             backedLayer.dispose()
             isDisposed = true
         }
     }
 
+    private fun disposeSkiaResources() {
```

Comment:
Can we make disposeSkiaResources/initSkiaResources symmetric? For better representation of the data flow. I.e. don't close picture/pictureRecorder here.

Also there is a PR, where I move contextHanlder from here. So we will have only `createRedrawer` + `syncSize` and `redrawer.dispose`, so we probably don't need separate functions.

Reply:
Removed these methods. It looks like after combining `ContextHandler` and `Redrawer` we won't need them.